src/tv2-common/blueprintConfig.ts

```diff
@@ -15,15 +15,14 @@ export interface TableConfigItemBreakers {
 	LoadFirstFrame: boolean
 }
 
-export interface TableConfigItemGFXTemplates {
+export interface TableConfigItemGfxTemplates {
```

Comment:
This should technically be in singular and not plural

Reply:
Agreed and fixed, for the new "old" interface as well.